src/woo_publications/publications/migrations/0014_mark_existing_uploads_complete.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.2.19 on 2025-02-12 16:15
+
+from django.db import migrations
+
+
+def mark_completed(apps, _):
+    Document = apps.get_model("publications", "Document")
+    Document.objects.exclude(lock="").update(upload_complete=True, lock="")
```

Comment:
Are we altering the "lock" value because it wasn't implemented correctly before? Or am I missing something?